force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -3951,6 +3951,22 @@
         <shortDescription>Release gates progress indicator when all releases activated</shortDescription>
         <value>All releases are active.</value>
     </labels>
+    <labels>
+        <fullName>stgReleaseGateEDASpring22Label</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>EDA Spring 22 release gate label</shortDescription>
+        <value>Spring&apos;22</value>
+    </labels>    
+    <labels>
+        <fullName>stgReleaseGateEDASpring22Description</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>EDA Spring 22 release gate description</shortDescription>
+        <value>There were no features gated for this release therefore all features were automatically released with Spring&apos;22</value>
```

Comment:
- [Future Enhancement] Parameterize release name to mitigate custom labels introduced per release.